conjure/controllers/deploystatus/gui.py

```diff
@@ -23,18 +25,30 @@ def __fatal(error):
     return __handle_exception('ED', Exception(error))
 
 
-def finish():
+def __handle_exception(tag, exc):
+    utils.pollinate(app.session_id, tag)
+    app.ui.show_exception_message(exc)
+
+
+def __wait_for_applications_exception(exc):
```

Comment:
fwiw, elsewhere we've been using partial(__handle_exception, "ED") instead of creating a whole wrapper. not a big deal though, just nice to be consistent